routes/kubernetes/kub.js

```diff
@@ -3,6 +3,26 @@ const express = require('express');
 const router = express();
 
 const k8sApi = k8s.Config.defaultClient();
+ 
+
+
+router.get('/openebsversion',(req, resp) => {
+  var openEbsVer="" ;
+
+  k8sApi.listNamespacedPod('openebs').then((res) => {
+    return new Promise(function (resolve, reject) {
+    for (i = 0; i < res.body.items.length; i++) {
```

Comment:
instead of for loop, you can directly use mayaapi-server deployment/pod to get the version

Reply:
In maya-api server volume name having like , `0.8.1 unreleased ` .